### DISCUSSION TITLE
Re-run the script for each flight?

### DISCUSSION THREAD

**Discussion by Motley-Brew** [jdholtz/auto-southwest-check-in#243] [Q&A] [answered]
First, thanks for writing this. Just getting started with it. I run it with user / pass (vs with confirmation #).

Basic question:  If I run it with user / pass (vs with conf #), will it check my account for new reservations every so often and add schedule them for check-in? If not, can I get to something like that by adding a script to crontab that kills the current process and re-starts it every night? 

I'm hoping this is something I can just let run in an Ubuntu VM (for now, until Docker works again), and it will monitor my account reservations in the background, checking me in, removing flights I've cancelled, etc.

THANKS!

**Comment by jdholtz** [comment, marked as answer]
Welcome!

> If I run it with user / pass (vs with conf #), will it check my account for new reservations every so often and add schedule them for check-in?

Yes, running the script with a username and password will check your account for new reservations every 24 hours (can be configured with the `retrieval_interval` config option) and schedule new flights it finds automatically.

> it will monitor my account reservations in the background, checking me in, removing flights I've cancelled, etc.

Yup, the script will check you in to any new or existing flights it sees, cancel check-ins for flights that no longer exist, and even monitor for lower fares on scheduled flights if you choose (by default, the script does).

**Reply by Motley-Brew** [reply]
Great, thanks!
